server/models/meshery_pattern.go

```diff
@@ -43,6 +47,7 @@ const (
 	DockerCompose DesignType = "Docker Compose"
 	K8sManifest   DesignType = "Kubernetes Manifest"
 	Design        DesignType = "Design"
+	OCI           DesignType = "OCI"
```

Comment:
@theBeginner86 this doesn't exactly compute given that "Design" and "OCI" are both assumed to be of the design format - just that one is directly yaml and the other needs to be unwrapped. Can you collapse these two and include logic that simply attempts to import either format (straight yaml or tarred yaml (in OCI))?

Reply:
You can reuse whatever test oras does for oci and an OpenAPI validation against Meshery's Design schema as format type tests. //@MUzairS15